com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/ObjectStoreFileStorage.java

```diff
@@ -137,6 +137,22 @@ private void putBlobWithRetries(Blob blob, int retries) {
         }
     }
 
+    private Blob getBlobWithRetries(FileEntry fileEntry, int retries) {
+        Blob blob = null;
+        for (int i = 1; i <= retries; i++) {
+            blob = blobStore.getBlob(container, fileEntry.getId());
+            if (blob != null) {
+                return blob;
+            }
+            LOGGER.warn(Messages.BLOB_STORE_GET_BLOB_FAILED, i, retries);
+            if (i == retries) {

```

Comment:
If you decide to do the below comment, this would be no longer needed, so you could remove this.

Reply:
Let's keep this so we don't sleep for longer and not needed time